poker.rb

```diff
@@ -0,0 +1,12 @@
+require './card'
+
+class Poker
+  def prompt_user
+    puts "Please enter a valid card (Ex. H2 is the Two of Hearts):"
+  end
+end
+
+game = Poker.new
+5.times do 
+  game.prompt_user
+end
```

Comment:
Looks like there's two things going on in this file -- the class deals with the user interface and the 4 lines of code below start the poker game. I think these two parts should be separated into different files. I'd also reconsider the name of the Poker class based what it does in its current state.